src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt

```diff
@@ -298,14 +299,30 @@ class XmppApi(
                 )
             }
             JibriMode.STREAM -> {
+                val rtmpUrl = if (startIq.streamId.startsWith("rtmp://", ignoreCase = true)) {
+                    startIq.streamId
+                } else {
+                    "$YOUTUBE_URL/${startIq.streamId}"
+                }
+                val viewingUrl = if (startIq.youtubeBroadcastId != null) {
+                    if (startIq.youtubeBroadcastId.startsWith("http", ignoreCase = true)) {
```

Comment:
I think valid youtube broadcast IDs can start with `http`

Reply:
If so, then we'll just use the URL as-is which I think would be fine.  But previously we always use the `youtubeBroadcastId` field to build a viewing URL via `http://youtu.be/${startIq.youtubeBroadcastId}`.

Reply:
My point is that if someone's youtube ID happens to be `httpXYZ123` we'll use that as a URL. Instead, we should use `https://youtu.be/httpXYZ123`

Reply:
Ah, I see.  I intentionally used only `http` to easily match both `http://` and `https://`, but I can explicitly check for either of those instead.